controllers/suite_test.go

```diff
@@ -49,33 +49,32 @@ package controllers
 //		[]Reporter{printer.NewlineReporter{}})
 //}
 //
-//var _ = BeforeSuite(func(done Done) {
-//	logf.SetLogger(zap.LoggerTo(GinkgoWriter, true))
+//var _ = BeforeSuite(func() {
+//	logf.SetLogger(zap.New(zap.WriteTo(GinkgoWriter), zap.UseDevMode(true)))
 //
 //	By("bootstrapping test environment")
 //	testEnv = &envtest.Environment{
-//		CRDDirectoryPaths: []string{filepath.Join("..", "config", "crd", "bases")},
+//		CRDDirectoryPaths:     []string{filepath.Join("..", "config", "crd", "bases")},
+//		ErrorIfCRDPathMissing: true,
 //	}
 //
-//	var err error
-//	cfg, err = testEnv.Start()
-//	Expect(err).ToNot(HaveOccurred())
-//	Expect(cfg).ToNot(BeNil())
+//	cfg, err := testEnv.Start()
+//	Expect(err).NotTo(HaveOccurred())
+//	Expect(cfg).NotTo(BeNil())
 //
 //	err = monitoringopenshiftiov1alpha1.AddToScheme(scheme.Scheme)
 //	Expect(err).NotTo(HaveOccurred())
 //
 //	// +kubebuilder:scaffold:scheme
 //
 //	k8sClient, err = client.New(cfg, client.Options{Scheme: scheme.Scheme})
-//	Expect(err).ToNot(HaveOccurred())
-//	Expect(k8sClient).ToNot(BeNil())
+//	Expect(err).NotTo(HaveOccurred())
+//	Expect(k8sClient).NotTo(BeNil())
 //
-//	close(done)
 //}, 60)
 //
 //var _ = AfterSuite(func() {
 //	By("tearing down the test environment")
 //	err := testEnv.Stop()
-//	Expect(err).ToNot(HaveOccurred())
+//	Expect(err).NotTo(HaveOccurred())
```

Comment:
Can we remove this code if we don't need it?

Reply:
It's from operator-sdk, which means it's gonna come back of we want it?